src/components/ProjectCard.tsx

```diff
@@ -114,114 +114,116 @@ export default function ProjectCard({
           : `/p/${_project?.handle}`
       }
     >
-      <div
-        style={{
-          borderRadius: radii.lg,
-          cursor: 'pointer',
-          overflow: 'hidden',
-
-          ...cardStyle,
-        }}
-        className="clickable-border"
-      >
-        <div style={{ marginRight: 20 }}>
-          <ProjectLogo
-            uri={metadata?.logoUri}
-            name={metadata?.name}
-            size={110}
-          />
-        </div>
+      <a>
```

Comment:
<a> will get the `href` from the parent `link`.

Example: https://nextjs.org/docs/api-reference/next/link